mod-orders/examples/claimingCollection.sample

```diff
@@ -0,0 +1,7 @@
+{
+  "claiming": [
```

Comment:
it is not clear that here is list of pieces.
Can rename for example to claimingPieceIds